views/index.erb

```diff
@@ -2,30 +2,14 @@
 <html lang=en-us>
 <head>
   <meta http-equiv="refresh" content="5">
-    <link rel="stylesheet" href="css/app.css" media="screen">
+  <link rel="stylesheet" href="css/app.css" media="screen">
+
 </head>
-<body>
+<body class="<%unless @alerts.empty? %> alert <% end %> ">
```

Comment:
only comment is that you could remove the spaces around `alert`

    <body class="<%unless @alerts.empty? %>alert<% end %>">